### ISSUE TITLE
PredictFractionalPeaksTest raises exception when run as part of CrystalTest unit test suite

### ISSUE THREAD

**Issue by DannyHindson** [mantidproject/mantid#27437]
The test_exec_without_hkl_range_uses_input_peaks test in the PredictFractionalPeaksTest test suite raises an unhandled exception when run as part of the full CrystalTest test suite causing the test suite to fall over (when running the test suite sequentially on Windows)

This is because an earlier test (PredictPeaksTest) sets the Q convention to "Crystallography":

 Kernel::ConfigService::Instance().setString("Q.convention",
                                                "Crystallography");

This applies a -1 to the HKL values that are loaded and causes the peaks list to be empty.

The test runs OK if run in isolation.

There are a few possible changes to address this: perhaps PredictPeaksTest should set the setting back to its original value when it completes. The PredictFractionalPeaks class maybe needs a m_qConventionFactor member similar to PredictFractionPeaks class? Also I think the test_exec_without_hkl_range_uses_input_peaks test should check if the frac_peaks PeaksWorkspace contains any peaks before trying to access the 0th peak

### Expected behaviour

test raises unhandled exception

### Actual behaviour

test should complete reporting fails\passes even if an earlier unit test has set Q.convention to Crystallography

### Steps to reproduce the behaviour

Run the CrystalTest unit test suite in full (takes 10-15 mins)

### Platforms affected

Have only tried on Windows. Ran the tests from within Visual Studio ie I didn't use the -j8 option which runs the tests in parallel

**Comment by DannyHindson** [comment]
Closing because I've entered another issue https://github.com/mantidproject/mantid/issues/27448 which includes some other failures when running the unit tests in a single thread
